doc/srv6/srv6-hld-v19.md

```diff
@@ -0,0 +1,552 @@
+            "path": [
+                "BABA:1001:0:10::",
+                "BABA:1001:0:20:F1::"
+            ]
+        },
+        "seg2": {
+            "path": [
+                "BABA:1001:0:30::",
+                "BABA:1001:0:40:F1::"
+            ]
+        }
+    }
+```
+
+**SRV6_LOCAL_SID_TABLE**
+
+Description: New table to hold local SID to behavior mapping
+
+Schema:
+
+```
+; New table
+; holds local SID to behavior mapping, allow 1:1 or n:1 mapping
+
+key = SRV6_LOCAL_SID:ipv6address
+; field = value
+block_len = blen             ; bit length of block portion in address, default 40
+node_len = nlen              ; bit length of node ID portion in address, default 24
+func_len = flen              ; bit length of function portion in address, default 16
+arg_len = alen               ; bit length of argument portion in address
+action = behavior            ; behaviors defined for local SID
+vrf = VRF_TABLE.key          ; VRF name for END.DT46, can be empty
+adj = address,               ; Optional, list of adjacencies for END.X
+policy = SRV6_POLICY.key     ; Optional, policy name for END.B6.ENCAP
+source  = address,           ; Optional, list of src addrs for encap for END.B6.ENCAP
+weight  = weight.value,      ; Optional, list of assigned weight
+
+For example:
+    "SRV6_LOCAL_SID" : {
+       	"BABA:1001:0:20:F1::" : {
+           "action": "end.dt46",
+           "vrf":  "VRF-1001"
+        },
+       	"BABA:1001:0:40:F1::" : {
+           "action": "end.dt46",
+           "vrf":  "VRF-1001"
+        },
+       	"BABA:1001:0:20:F2::" : {
+           "action": "end.x",
+           "adj": [
+                BABA:2001:0:10::1, 
+                BABA:2001:0:10::2
+            ],
+            "weight": "1,2"
+        },
+       	"BABA:1001:0:20:F3::" : {
+           "action": "end.b6.encap",
+           "segment": "seg1"
+           "source": "A::1"
+        }
+    }
+```
+
+**SRV6_POLICY_TABLE**
+
+Description: New table that stores SRv6 policy .
+
+Schema:
+
+```
+; New table
+; holds SRv6 policy
+
+key = SRV6_POLICY:policy_name
+
+; field = value
+endpoint = address             ; 
+segment = SRv6_SID_LIST.key,   ; List of segment names
+weight  = weight.value,        ; List of assigned weight value
+
+For example:
+    "SRV6_POLICY": {
+        "policy1": {
+            "segment": "seg1, seg2",
+            "weight": "1, 2"
+            },
+        "policy2": {
+            “endpoint": "BABA:1001:0:40::1"
+            "segment": "seg1"
+        }
+    }
+```
+
+**SRV6_STEER_MAP**
+
+Description: New table that stores prefix to policy mapping .
+
+Schema:
+
+```
+; New table
+; holds prefix to SRv6 SID list encapsulation mapping
+
+key = SRV6_STEER:VRF_NAME:prefix
+                               ; Prefix to be steered
+; field = value
+policy = SRV6_POLICY.key       ; Policy to steer the prefix
+source  = address              ; Source addresses for encapsulation
+
+For example:
+    "SRV6_STEER": {
+        "Vrf-red|11.11.11.0/24": {
```

Comment:
I assume, the keys SRV6_STEER:VRF_NAME:prefix and ROUTE_TABLE:VRF_NAME:prefix are always the same, right? if yes, can we just have steer name as a key in this table (SRV6_STEER) and refer from ROUTE_TABLE?

Reply:
has reach the consensus in the review meeting and keep as it was.